intern/cycles/blender/blender_sync.h

```diff
@@ -97,6 +97,7 @@ class BlenderSync {
 	/* sync */
 	void sync_lamps(bool update_all);
 	void sync_materials(bool update_all);
+	void sync_light_linking();
```

Comment:
cycles code style suggests that this should be  `void sync_light_linking(void);` by the way ;) [closing this pull request though]